gammapy/data/data_store.py

```diff
@@ -292,7 +289,7 @@ def get_observations(self, obs_id=None, skip_missing=False, required_irf="all"):
             )
 
         if obs_id is None:
-            obs_id = self.obs_table["OBS_ID"].data
+            obs_id = np.unique(self.hdu_table["OBS_ID"].data)
```

Comment:
Make this an 

```
@property
def obs_ids(self):
     return np.unique(self.hdu_table["OBS_ID"].data)
```

?

Depending on whether you consider the data store mutable or not, this could also be a `lazyproperty` then.

Reply:
Done